services/user.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ObjectDoesNotExist
+
+
+User = get_user_model()
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str | None = None,
+    first_name: str = "",

```

Comment:
`None` is preferable default value for optional parameters + here you need to add annotation that it is optional same as above